action.yml

```diff
@@ -1,6 +1,6 @@
 name: 'StaleSweeper'
 description:
-  'Automatically closes discussions that have not had recent interaction'
+  'Automatically closes stale discussions to declutter your GitHub repository.'
```

Comment:
Hmm, I guess not your GitHub repository per say? More like the discussion fora, but maybe it's fine

Reply:
Let's change it to something as clear as possible 👍🏻 

However, to be honest, never heard the word fora before Denmark, do you mean 'forum'? 🤔

Reply:
Fora is forum in plural 😅

Reply:
Dejligt 😅